internal/repotools/repo_root.go

```diff
@@ -0,0 +1,56 @@
+package repotools
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// FindRepoRoot returns the absolute path to the root directory of the
+// repository, or error. If the dir passed in is a relative path it will be
+// used relative to the current working directory of the executable.
+func FindRepoRoot(dir string) (string, error) {
```

Comment:
Should `makerelative` command be updated  to share the same findRepoRoot logic?

Reply:
good point, i missed that